README.md

```diff
@@ -92,7 +92,7 @@ You can then load the module in GHCi, and run it:
 <img width="307" src="https://github.com/hedgehogqa/haskell-hedgehog/raw/master/img/hedgehog-logo-grey.png" />
 
  [hackage]: http://hackage.haskell.org/package/hedgehog
- [hackage-shield]: https://img.shields.io/badge/hackage-v1.0.1-blue.svg
+ [hackage-shield]: https://img.shields.io/badge/hackage-v1.0.2-blue.svg
```

Comment:
```suggestion
 [hackage-shield]: https://img.shields.io/hackage/v/hedgehog.svg?style=flat
```

We actually fixed our .cabal file so the version is on the same line as the `version:` text, so the shields js works properly now. Might as well make this automatic.